Global.-1.ttslua

```diff
@@ -1173,19 +1168,21 @@ end--function GameLoop END
 ---------------------------------------------------------------------------------------------------------------------------
 --[[This function is called when a player clicks the UNO button]]
 function Player_Calling_UNO(player,b,button)
-    
+    --If uno_Called is set to true, do nothing because the button has already been pressed
+    if(uno_Called) return end
```

Comment:
just a small syntax error here. you forgot to add a 'then' to your if-statement

Reply:
I could have sworn I fixed that. Anyway, I don't think I can edit it since the pull request is closed.

Reply:
Yeah, It's no big deal. I just noticed it after i merged and fixed it.

Reply:
Okay sounds good.